components/progress-bar/examples/00-default.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import { ProgressBar } from '../src';
+
+export default () => {
+	return (
+		<>
+			<ProgressBar valuenow="45" />
+			<ProgressBar valuenow="74" skinny />
```

Comment:
Wonder if we should pass a non-string (int/float) value here...

`<ProgressBar valuenow={45} />`

Does that even matter? How does that affect the proptype? (currently `PropTypes.number`)

@jaortiz ?